apps/onboarding/src/migrations/1607954006850-session.ts

```diff
@@ -0,0 +1,50 @@
+import {MigrationInterface, QueryRunner, Table} from "typeorm"
+
+/* tslint:disable */
+export class session1607954006850 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<any> {
+        await queryRunner.createTable(
+          new Table({
+            name: 'session',
+            columns: [
+              {
+                name: 'id',
+                type: 'uuid',
+                // isPrimary: true,
+              },
+              {
+                name: 'externalAccount',
+                type: 'varchar',
+                isNullable: false,
+              },
+              {
+                name: 'meta',
+                type: 'json',
+                isNullable: true
+              },
+              {
+                name: 'createdAt',
+                type: 'timestamp',
+                isNullable: false
+              },
+              {
+                name: 'expiredAt',
+                type: 'timestamp',
+                isNullable: true
+              },
+              {
+                name: 'completedAt',
+                type: 'timestamp',
+                isNullable: true
+              }
+            ]
+          }),
+          false,
```

Comment:
```suggestion
          true,
```

If we set this to `true` the migration will run both in our already setup databases in the running clusters, and [short-circuit the table creation](https://github.com/typeorm/typeorm/blob/51b2a63d/src/driver/postgres/PostgresQueryRunner.ts#L298-L302). But the migration will get marked as completed, and in new environments where table doesn't exist, it will create it.